echo.js

```diff
@@ -1,7 +1,9 @@
 function echo(){
-  // ...
+  for (var i = 0, ilen = arguments.length; i < ilen; i++) {
+  	console.log(arguments[i]);
```

Comment:
This warning is a fluke – will try and get it sorted out for the next assignment. Sorry for any confusion!